### DISCUSSION TITLE
kairship/intro/faq

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2301] [Announcements]
# kairship/intro/faq

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/kairship/intro/faq.html

**Comment by liyuerich** [comment]
Karmada 的版本是多少？能否指定版本？是否可以升级？

当前默认版本 v1.4.0，支持用户自主升级

当前版本已经不是v1.4.0了吧？ 能否给怎么看版本的方法？
